cadquery/occ_impl/shapes.py

```diff
@@ -2704,6 +2705,24 @@ def normalAt(self, locationVector: Optional[Vector] = None) -> Vector:
 
         return Vector(vn)
 
+    @normalAt.register
+    def normalAt(self, u: Real, v: Real) -> Tuple[Vector, Vector]:
+        """
+        Computes the normal vector at the desired location in the u,v parameter space.
+
+        :returns: a vector representing the normal direction and the position
+        :param u: the u parametric location to compute the normal at.
+        :type u: real.
```

Comment:
I think the type specifier can be omitted from docstring because the type hint is already defined.